packages/auth/lib/AccessToken.js

```diff
@@ -45,34 +50,47 @@ const getHmac = (payload, key) => {
     .digest('hex')
 }
 
-const getPayload = ({ userId, scope, expiresAt }) => {
-  const expiresAtFormatted = moment(expiresAt).format(DATE_FORMAT)
-  return `${userId}/${scope}/${expiresAtFormatted}`
-}
+const getPayload = ({ userId, scope, expiresAt }) => `${userId}/${scope}/${expiresAt}`
 
 const generateForUser = (user, scope) => {
   const key = user.accessKey || user._raw.accessKey
   if (!key) {
     throw new MissingKeyError()
   }
-  const scopeConfig = getScopeConfig(scope)
+  const { ttlDays, expireAtFormat = 'YYYY-MM-DD' } = getScopeConfig(scope)
+
   const payload = getPayload({
     userId: user.id,
     scope,
-    expiresAt: moment().add(scopeConfig.ttlDays, 'days')
+    expiresAt: moment().add(ttlDays, 'days').format(expireAtFormat)
   })
+
   return base64u.encode(`${payload}/${getHmac(payload, key)}`)
 }
 
+const issueForUser = (issuer, user, scope) => {
```

Comment:
not happy with the naming here either. `generateForUser` and `issueForUser` just means the same and are not self explaining. How about `generateForUser` and `generateByUserForUser` or `generateForUserNoAuthChecks` and `generateForUser`.

Reply:
Renamed to `generateForUserByUser(user, scope, me,...)`